graphs/possible_bipartition.py

```diff
@@ -1,12 +1,42 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(NE)
+        Space Complexity: O(n)
     """
```

Comment:
I think this is O(N + E) for time complexity because I think you hit every node and check every edge that node has.